lib/opam.ml

```diff
@@ -216,3 +216,25 @@ module Extra_field = struct
 
   let get t opam = OpamFile.OPAM.extended opam t.name t.from_opam_value
 end
+
+module Pos = struct
+  open OpamParserTypes.FullPos
```

Comment:
Yeah I think it's only for the record fields, makes the code more concise since there's plenty of record access here, happy to remove it as I am myself not a huge fan of global opens!

Reply:
I concur, stating the labels explicitly (or through an module alias to avoid repeating `OpamParserTypes.FullPos` everywhere) could make it more readable as it is clearer what it is used for.